melos.yaml

```diff
@@ -82,10 +83,16 @@ scripts:
     description: Deploy cairo contracts
     run: |
       starkli deploy $HELLO_CLASS_HASH --salt $SALT 0x0
-  hello:get_name:
-    description: Call hello get_name
+
+  contracts:invoke:
+    description: creating invoke tx for tests with hash 0x03b2911796e0024f9e23d7337997538058eca267d5ddaa582d482cbe1fb64897 and deploy_account tx with hash 0x055ba13c33a12506d2eab8dfbc618a8ce0d247c24959a64ee18fbf393c873b83
     run: |
-      starkli call $HELLO_CONTRACT_ADDRESS get_name
+      starkli invoke $HELLO_CONTRACT_ADDRESS set_name 0x0 
+      # should create tx hash 0x029583643cd8932f1955bf28bfebf4c907b13df1e5c2d202b133cfbf783697a2
+      
+      yes | cp .starkli/account_for_devnet_setup.json .starkli/account_1.json
+      starkli invoke 0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7 transfer 0x016a0d7df981d681537dc2ce648722ff1d1c2cbe59412b492d35bac69825f104 0x100000000000000000 0x0
```

Comment:
It should be easier to read with `eth` instead of `0x49D...`

Reply:
done